src/components/include/utils/threads/thread.h

```diff
@@ -193,7 +221,7 @@ class Thread {
    * @return true if the thread has been started, and not yet stopped.
    */
   bool is_running() const {
-    return isThreadRunning_;
+    return (kThreadStateRunning == thread_state_);
```

Comment:
@anosach-luxoft, fixed in [ce7130e](https://github.com/smartdevicelink/sdl_core/pull/707/commits/ce7130e0fa97a34d5fb143ff0764557cee82eb68)